src/xpath.js

```diff
@@ -0,0 +1,17 @@
+define([
+    'xpath',
+    'xpathmodels'
+], function (
+    xpath,
+    makeXPathModels
+) {
+    /* global xpathmodels */
+    /*jshint -W020 */
+    xpathmodels = makeXPathModels();
+    /*jshint +W020 */
+    
+    return {
+        xpath: xpath,
+        xpathmodels: xpathmodels
```

Comment:
what do you think of renaming this to `models` so the reference elsewhere would be `xpath.models`?